update readme (#25)
qwint/pseudoregalia-archipelago: README.md

Reviewer: we can mention that outfits are shuffled as well in the header line

# Mod Installation

> Note: If you are installing on Steam Deck, please read [the Steam Deck installation instructions](https://github.com/qwint/pseudoregalia-archipelago/blob/main/docs/steam_deck_installation.md) instead.

Reviewer: the downpatch info should be added to steam deck instructions file as well

> 1. [Go to the Releases page](https://github.com/qwint/pseudoregalia-archipelago/releases/latest) and download the latest version of `pseudoregalia-archipelago.zip` and `pseudoregalia.apworld`.

Reviewer: ```suggestion
1. [Go to the Releases page](https://github.com/qwint/pseudoregalia-archipelago/releases/latest) and download the latest version of `pseudoregalia-archipelago.zip`.
```
since the mod doesn't actually need the apworld and we talk about installing the apworld in the generating section i think we can just remove this from here

2. In Steam/steamapps/common, make a copy of your Pseudoregalia folder and rename it "Pseudoregalia_AP" (or whatever else you like).
   * You can add the new `pseudoregalia.exe` as a "Non-Steam Game" in Steam to access it easily from Steam and to maintain Steam Input compatibility (which may be required for some controllers)

>    * If you would like to play on full gold version, down patch using the beta code `fullgoldjump` in Steam before copying the folder. You can make an AP folder for both versions if you want to be able to play on both.

Reviewer: ```suggestion
   * If you would like to play on full gold version, downpatch using the beta code `fullgoldjump` in Steam before copying the folder. You can make an AP folder for both versions if you want to be able to play on both.
```
no strong feelings about this but seeing them as separate words genuinely threw me lol (hyphenated is an ok middleground if you like that better than a single word)

3. Extract the contents of `pseudoregalia-archipelago.zip` directly into "Pseudoregalia_AP".

> ![](https://i.imgur.com/SGPm9oq.jpg)

Reviewer: we really need to update this image lol, you get none of these options anymore

![](https://i.imgur.com/9IedtY4.jpg)

> 2. Place `pseudoregalia.apworld` into Archipelago/custom_worlds.

Reviewer: ```suggestion
2. [Go to the Releases page](https://github.com/qwint/pseudoregalia-archipelago/releases/latest) and download the latest version of `pseudoregalia.apworld` and either double click to open the file with Archipelago, or manually place `pseudoregalia.apworld` into `Archipelago/custom_worlds` to install.
```
